### PR TITLE
Test-harness multiple provisioners

### LINKED ISSUE #1317: Add staking smoke tests
**Describe what you want implemented** 
Following is a laundry list of checks to be performed to ensure the stability of the network locally:
[ ] lower the maturity period on the stake contract down to 5 blocks
[ ] create 10 stakes
[ ] run a local cluster with 10 Provisioners

The monitoring should focus on memory usage, block time, eventual errors of the consensus, provisioner synchronization

**Describe "Why" this is needed**
This is needed as a first step toward the full multi-provisioner network cluster.

**Describe alternatives you've considered**
Create a genesis block with multiple provisioners already embedded. Without lowering the maturity period, this would only enable testing of the consensus, as any stake would require ~20h to become active. Eventually we will have to do both, but not immediately.

**Additional context**

### PR DESCRIPTION
**Pull request by herr-seppia**
- [x] Add reward generator to the rusk command
- [x] ~Send transfer from faucet to provisioner~
- [x] ~Stake from each provisioner~
- [x] Include more stakers in the genesis block

Resolves #1317

### CODE REVIEW

File: pkg/config/consts.go [hunk 1]
```diff
@@ -26,7 +26,7 @@ const (
 	MaxInvBlocks = 500
 
 	// Protocol-based consensus step time.
-	ConsensusTimeOut = 20 * time.Second
+	ConsensusTimeOut = 5 * time.Second
```
Comment (autholykos) [inline pkg/config/consts.go:29]: back to 5s?

Comment (herr-seppia) [reply pkg/config/consts.go:29]: Yes, this was the timeout prior to increase block time to 15secs

File: pkg/core/data/ipc/transactions/provider.go [hunk 2]
```diff
@@ -122,6 +122,8 @@ type executor struct {
 
 // VerifyStateTransition see also Executor.VerifyStateTransition.
 func (e *executor) VerifyStateTransition(ctx context.Context, calls []ContractCall, blockGasLimit, blockHeight uint64) error {
+	time.Sleep(1 * time.Second)
```
Comment (autholykos) [inline pkg/core/data/ipc/transactions/provider.go:125]: What are the sleeps for?

**Review by autholykos — COMMENTED** [review]
I am fuzzy about the `sleep` before the state transition operations. The rest looks fine

**Review by autholykos — APPROVED** [review]
LGTM (sleeps will be tackled in a followup issue)

**Pull request merged**